include/deal.II/numerics/vector_tools_evaluate.h

```diff
@@ -547,8 +560,9 @@ namespace VectorTools
                                           typename VectorType::value_type>>> &)
     {
       (void)evaluation_flags;
+      (void)first_selected_component;
 
-      Assert(n_components == 1,
+      Assert(n_components == 1 && first_selected_component == 0,
```

Comment:
That's the only place where you use `first_selected_component`, right? Did I overlook something?

Reply:
It is used here:

https://github.com/dealii/dealii/blob/29d31b186cdbf6e0e6bc0ad8f309753cb97f6d30/include/deal.II/numerics/vector_tools_evaluate.h#L446-L453